specification/sql/resource-manager/Microsoft.Sql/preview/2023-05-01-preview/IPv6FirewallRules.json

```diff
@@ -0,0 +1,317 @@
+      "get": {
+        "tags": [
+          "IPv6FirewallRules"
+        ],
+        "description": "Gets a list of IPv6 firewall rules.",
+        "operationId": "IPv6FirewallRules_ListByServer",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the list of IPv6 firewall rules.",
+            "schema": {
+              "$ref": "#/definitions/IPv6FirewallRuleListResult"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 ManagedInstanceStoppingOrStopped - Conflicting operation submitted while instance is in stopping/stopped state\n\n * 400 ManagedInstanceStarting - Conflicting operation submitted while instance is in starting state\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription.\n\n * 404 ResourceNotFound - The requested resource was not found."
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List IPv6 Firewall Rules": {
+            "$ref": "./examples/IPv6FirewallRuleList.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/ipv6FirewallRules/{firewallRuleName}": {
+      "get": {
+        "tags": [
+          "IPv6FirewallRules"
+        ],
+        "description": "Gets an IPv6 firewall rule.",
+        "operationId": "IPv6FirewallRules_Get",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "firewallRuleName",
+            "in": "path",
+            "description": "The name of the firewall rule.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the specified IPv6 firewall rule.",
+            "schema": {
+              "$ref": "#/definitions/IPv6 FirewallRule"
```

Comment:
@jeremyfrosti @rkmanda @ericshape @ms-henglu @jacobopac FYI this is a regression from #26303 - can we ensure that fix is included in this (and any future) API versions, since spaces are not valid identifiers